cockroachcloud/quickstart.md

```diff
@@ -6,372 +6,85 @@ referral_id: docs_quickstart_free
 docs_area: get_started
 ---
 
-This page shows you how to get started with CockroachDB quickly. You'll create a CockroachDB cluster, and then insert and read some sample data from a sample application.
+This page shows you how to get started with CockroachDB quickly. You'll use [`ccloud`](ccloud-get-started.html) to create a CockroachDB cluster, and then insert and read some sample data from a Node.js sample application.
 
-<div class="filters clearfix">
-    <button class="filter-button page-level" data-scope="serverless"><strong>Use a cloud cluster</strong></button>
-    <button class="filter-button page-level" data-scope="local"><strong>Use a local cluster</strong></button>
-</div>
-
-<section class="filter-content" markdown="1" data-scope="serverless">
-
-{{ site.data.products.serverless }} is the easiest way to get started with CockroachDB Cloud. Follow these steps to create a new {{ site.data.products.serverless }} cluster.
-
-To get started with {{ site.data.products.db }} using {{ site.data.products.dedicated }} clusters, see [Quickstart with {{ site.data.products.dedicated }}](quickstart-trial-cluster.html).
-
-</section>
-{% comment %} End serverless {% endcomment %}
-
-<section class="filter-content" markdown="1" data-scope="local">
-
-Follow these steps to get started with CockroachDB using a {{ site.data.products.core }} cluster.
-
-</section>
-{% comment %} End local {% endcomment %}
-
-<section class="filter-content" markdown="1" data-scope="serverless">
-
-<h2>Choose your installation method</h2>
-
-You can install a {{ site.data.products.serverless }} cluster using either the CockroachDB Cloud Console, a web-based graphical user interface (GUI) tool, or <code>ccloud</code>, a command-line interface (CLI) tool.
-
-<div class="filters clearfix">
-    <button class="filter-button page-level" data-scope="console">Use the Cloud Console (GUI)<strong></strong></button>
-    <button class="filter-button page-level" data-scope="ccloud"><strong>Use <code>ccloud</code> (CLI)</strong></button>
-</div>
-
-<section class="filter-content" markdown="1" data-scope="console">
-
-Follow these steps to create a {{ site.data.products.serverless }} cluster using the CockroachDB Cloud Console, a web based GUI.
-
-## Create a free cluster
-
-{% include cockroachcloud/quickstart/create-a-free-cluster.md %}
-
-## Create a SQL user
-
-{% include cockroachcloud/quickstart/create-first-sql-user.md %}
-
-## Connect to the cluster
-
-</section>
-{% comment %} End console {% endcomment %}
-
-<section class="filter-content" markdown="1" data-scope="ccloud">
-
-Follow these steps to create a {{ site.data.products.serverless }} cluster using the <code>ccloud</code> CLI tool.
+{{ site.data.products.serverless }} is the easiest way to get started with CockroachDB. You can also [start a local cluster](../{{site.versions["stable"]}}/start-a-local-cluster.html) or [start a {{ site.data.products.dedicated }} cluster](quickstart-trial-cluster.html).
```

Comment:
Not sure if we need this anymore. You can move the links to local cluster/dedicated cluster creation to the Learn more section